packages/flutter_tools/test/commands.shard/hermetic/build_ios_test.dart

```diff
@@ -176,7 +182,13 @@ void main() {
   }
 
   testUsingContext('ios build fails when there is no ios project', () async {
-    final BuildCommand command = BuildCommand();
+    final BuildCommand command = BuildCommand(
+      androidSdk: FakeAndroidSdk(),
+      buildSystem: TestBuildSystem.all(BuildResult(success: true)),
+      fileSystem: MemoryFileSystem.test(),
```

Comment:
do we need the filesystem override on line 199? And how does the global `XcodeProjectInterpreter` get called?

Reply:
In general, are we able to remove any of the context overrides from the tests after this change?

Reply:
Yes we still need those overrides because `BuildIOSCommand` is not yet migrated. As for the `XcodeProjectInterpreter` comes all the way from `lib/src/ios/mac.dart`

Reply:
ahhh, ok

Reply:
there are no overrides throughout all the test files here that can be removed?

Reply:
No 😕, wanted to remove the fs but it is used on `FlutterCommand`

Reply:
:'( ok, thanks for checking